lib/internal/bootstrap/node.js

```diff
@@ -527,6 +528,33 @@
     });
   }
 
+  function setupQueueMicrotask() {
+    const { queueMicrotask } = NativeModule.require('internal/queue_microtask');
+    Object.defineProperty(global, 'queueMicrotask', {
+      get: () => {
+        process.emitWarning('queueMicrotask() is experimental.',
+                            'ExperimentalWarning');
+        Object.defineProperty(global, 'queueMicrotask', {
+          value: queueMicrotask,
+          writable: true,
+          enumerable: false,
+          configurable: true,
```

Comment:
☝️ Might put a note that "enumerable" should be flipped to `true` when out of experimental. It's that way for other timer and prompt methods _(Safari Technology Preview has it enumerable for `queueMicrotask` too)_. Not sure in the spec where it's specified though.

Reply:
it's specified with webidl so you're right it should be enumerable. I'll change it.

Reply:
Experimental things are kept non-enumerable so they aren't tripped by folks iterating the objects/inspection.

Reply:
once it's accessed it should be the correct descriptor. notice there are two levels of defineProperty here